dashboard/src/test/java/com/google/cloud/tools/opensource/dashboard/DashboardTest.java

```diff
@@ -138,6 +139,10 @@ public void testDashboard()
         }
       }
 
+      // TODO these should all be separate tests for the different components
+      Node linkage = document.query("//pre[@id='static_linkage_errors']").get(0);
+      Assert.assertTrue(linkage.getValue().startsWith("guava-20.0.jar (0 errors)"));
```

Comment:
Truth shall <del>set you free</del> give you better error messages.

```suggestion
      assertThat(linkage.getValue()).startsWith("guava-20.0.jar (0 errors)");
```